### PR TITLE
Added new fork 7 for EON 1.5 disabling of backward transfers

### PR DESCRIPTION
**Pull request by alsala**
Introduce a EON 1.5 hardfork that will disable backward transfer requests to the mainchain

https://horizenlabs.atlassian.net/browse/EON-1913

### CODE REVIEW

File: node/src/main/java/io/horizen/eon/forks/F7Fork.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import io.horizen.utils.Pair;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * EON fork 7 (introduced in Version 1.5.0)
+ * Disables the backward transfers submission as part of the strategy which stops any cross-chain transfer operation between Mainchain and Sidechain with a view to EON2 migration.
+ */
+public class F7Fork extends EONFork {
```
Comment (saratnt) [inline node/src/main/java/io/horizen/eon/forks/F7Fork.java:15]: Shouldn't be added to EonForkConfigurator class?

Comment (alsala) [reply node/src/main/java/io/horizen/eon/forks/F7Fork.java:15]: Fixed

**Review by saratnt — APPROVED** [review]

**Review by paolocappelletti — APPROVED** [review]

File: node/pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <maven.compiler.target>11</maven.compiler.target>
     <maven.compiler.plugin>3.8.1</maven.compiler.plugin>
     <maven.dependency.plugin>3.1.1</maven.dependency.plugin>
-    <sidechain.sdk.version>0.12.0</sidechain.sdk.version>
+    <sidechain.sdk.version>0.13.0-SNAPSHOT</sidechain.sdk.version>
```
Comment (paolocappelletti) [inline node/pom.xml:14]: please update also  the pom version
1.4.0 -> 1.5.0-SNAPSHOT
the changelog: CHANGELOG.md
and the doc/release folder

Comment (alsala) [reply node/pom.xml:14]: Ok, done

**Pull request merged**